# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: js/src/jit-test/tests/ion/bug1572051.js
# Commit: bde81f964234
# Full Hash: bde81f9642344b7bc2fab62380771c670138aaac
# Author: Jan de Mooij <jdemooij@mozilla.com>
# Date: 2019-08-08 21:45:31
# Description:
#   Bug 1572051 - Add TypeMonitorMagicValue and use it in DoTypeMonitorFallback and TypeMonitorResult. r=iain
#   
#   We did a bailout from Ion with a MagicValue(JS_OPTIMIZED_OUT) on top of the stack, then passed that
#   value to TypeMonitorResult and asserted in addMonitorStubForValue because we expected a different
#   kind of magic value.
# ==============================================================================

diff -r 7d7830b79440 -r bde81f964234 js/src/jit-test/tests/ion/bug1572051.js
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/js/src/jit-test/tests/ion/bug1572051.js	Wed Aug 07 19:04:38 2019 +0000
@@ -0,0 +1,6 @@
+evaluate(`
+  for (var i = 0; i < 2000; i++) {
+    Array(Math, {});
+    bailout();
+  }
+`);